DXMainClient/Domain/Multiplayer/CnCNet/MapSharer.cs

```diff
@@ -207,62 +208,62 @@ private static byte[] UploadFiles(string address, List<FileToUpload> files, Name
         {
             //try
             //{
```

Comment:
I'd say remove the comments here. They can be found in history anyways, they look ugly here.